### PR TITLE
Plex improvements

### PR DESCRIPTION
**Pull request by JesseWebDotCom**
Updated documentation for the following pull request:
https://github.com/home-assistant/home-assistant/pull/6054

### CODE REVIEW

**Comment by JesseWebDotCom** [comment]
I assume the compatibility table markdown will render correctly as it does below because it does not when viewing here:
https://github.com/JesseWebDotCom/home-assistant.github.io/blob/c1f4b94beeadd6e33c6027d43fff20bc4af5ed00/source/_components/media_player.plex.markdown

### Compatibility
| Client  | Limitations |
| -- | --- |
| Any (when all controls disabled) | A stop button will appear but is not functional. |
| Any (when casting) | Controlling playback will work but with error logging. |
| Any (remote client) | Controls disabled. |
| Apple TV (PlexConnect) | Controls disabled.  Music does not work. |
| iOS  | None |
| NVidia Shield | Mute disabled. Volume set below 2 will cause error logging. Controlling playback when the Shield is both a client and a server will work but with error logging |
| Plex Web | None |
| Tivo Plex App  | Only play, pause, stop/off controls enabled |

**Comment by JesseWebDotCom** [comment]
The associated pull request was approved.  Do I need to do anything else to get this document approved (like click close and comment)?  Thanks

**Comment by JesseWebDotCom** [comment]
Robbie - I spotted and fixed a bug (link below) but I don't believe that will get merge in unless someone with your access does something:
https://github.com/JesseWebDotCom/home-assistant/commit/3bb8fc080d47904b2f8da4f299532b86c91066e1

If you are able to merge the change above, then yes the documentation is up to date.  If not, I'd have to comment out documentation on the show all controls feature (as it won't work without the fix above).

Also, do you have a way to ensure the markdown looks good?  That's my only concern about the documentation.  Thanks

**Comment by robbiet480** [comment]
@JesseWebDotCom you need to stage that change off of an up to date `dev` branch to get it merged in. 

We can merge these docs as is since that change looks valid.

Did you try running `rake preview` to check the site out locally? If you can't/don't want to I can try and report back results.

**Comment by JesseWebDotCom** [comment]
Have not (wasn't aware and am still learning).  I can do the first thing you mentioned if you can provide some guidance on the steps.

**Comment by robbiet480** [comment]
Actually, I gotchu

**Comment by robbiet480** [comment]
Hit [this link](https://github.com/robbiet480/home-assistant/pull/new/new-plex-fix) and you should be prompted to open a pull request with your commit.

**Comment by JesseWebDotCom** [comment]
Done, thanks - When I get more experienced, I'll definitely help create a step by step for newbie

**Comment by robbiet480** [comment]
@JesseWebDotCom Now that that's done your existing `dev` branch is a bit screwy because it's behind. If you give me access to your repo for a few minutes I can get it fixed for you.

**Comment by robbiet480** [comment]
Just add my name [here](https://github.com/jessewebdotcom/home-assistant/settings/collaboration)

**Comment by JesseWebDotCom** [comment]
done

**Comment by robbiet480** [comment]
Your dev branch is now fixed. Next time that you want to submit work to Home Assistant make a branch off of dev branch. Makes everything a lot cleaner :)

You can remove me from that settings screen now.

**Comment by JesseWebDotCom** [comment]
Thanks - if you have some newbie study materials, send them my way

**Comment by robbiet480** [comment]
[Here's the general flow](https://home-assistant.io/developers/development_submitting/)

**Comment by JesseWebDotCom** [comment]
Thanks - still want me to preview if my markdown formatting was good?

File: source/_components/media_player.plex.markdown [hunk 1]
```diff
@@ -47,13 +51,81 @@ In case [discovery](/components/discovery/) does not work (GDM disabled or non-l
...
+
+#### Video
+
+| Service data attribute | Optional | Description | Example |
+| ---------------------- | -------- | ----------- | ----------- |
+| `entity_id` | no | `entity_id` of the client | media_player.theater_plex |
+| `media_content_id` | no | Quote escaped JSON with `library_name`, `video_name`, `shuffle` (0 or 1). | { \\"library_name\\" : \\"Adult Movies\\", \\"video_name\\" : \\"Blade\\", \\"shuffle\\": \\"0\\" } |
+| `media_content_type` | no | Type of media to play, in this case `VIDEO` | VIDEO |
+
+### Compatibility
```
Comment (robbiet480) [inline source/_components/media_player.plex.markdown]: This is the only section of Markdown that is broken. Will reformat.

**Comment by robbiet480** [comment]
Thanks! 🐬 🍪 💯

**Comment by robbiet480** [comment]
@JesseWebDotCom Seems your long nightmare is now over 😹

**Comment by JesseWebDotCom** [comment]
That's the price you pay for trying to help others (as you already know)

**Comment by JesseWebDotCom** [comment]
Let me know if you need an updated mini changelog or anything else for the official release

**Pull request merged**